packages/packagers/js/src/ScopeHoistingPackager.js

```diff
@@ -130,7 +130,7 @@ export class ScopeHoistingPackager {
 
       let [content, map, lines] = this.visitAsset(asset);
       if (sourceMap && map) {
-        sourceMap.addBufferMappings(map.toBuffer(), lineCount);
+        sourceMap.addBuffer(map.toBuffer(), lineCount);
```

Comment:
Should we use `sourceMap.addSourceMap(map, lineCount)` here? I think it should improve performance a bit.

Reply:
Oh yes definitely